### ISSUE TITLE
Dropdown: Narrator does not announces the Role of the Combo box in Fabric Dropdown component

### ISSUE THREAD

**Issue by sulabh-msft** [OfficeDev/office-ui-fabric-react#6892]
### Environment Information 
 
 
- __Package version(s)__: (fill this out) 
- __Browser and OS versions__: (fill this out if relevant)

### Please provide a reproduction of the bug in a codepen:
 Bug Description:
Check out Keros! - Identify accessibility bugs before check-in and make bug fixing faster and easier. 

#### Repro Steps:
1. Launch application using above URL. (https://fabricweb.z5.web.core.windows.net/oufr/6.94.0/#/examples/dropdown)
2. Start Narrator.
3. Navigate to the "Basic uncontrolled example:" example
5. Verify Narrator not announces the Role of the "Basic uncontrolled example" component ( Dropdown)
 
#### Actual:
Narrator does not announces the Role of the Dropdown
 
Same issue is observed in "Chrome+NVDA" configuration.
 
#### Expected:
Narrator should announce the Role of the "Basic uncontrolled example" as "Combo box"
 
#### Recommendation:
https://microsoft.sharepoint.com/teams/msenable/mas/Pages/browse-fixes.aspx

#### User Impact:
ScreenReader-only user will get difficulty to understand if Narrator not announces the role of the dropdown.
 
#### MAS Reference:
MAS1.3.1-
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={54f28d1f-a2d1-4dcd-84e1-5c9b87e8aba4}

 

![image](https://user-images.githubusercontent.com/13728069/47702183-29adac00-dc42-11e8-91f2-879ed28336b4.png)

#### Priorities and help requested:
 
Are you willing to submit a PR to fix? (No)
 
Requested priority: (Blocking, High, Normal, Low) 
 Normal

Products/sites affected: (if applicable)
Azure Devops, Project overview page

**Comment by JasonGore** [comment]
I'm going to mark this as a dupe of #6326 and have added a reference to this issue there.
